### PR TITLE
Loading indicator implementation

### PR DESCRIPTION
**Pull request by dparikh**

### CODE REVIEW

File: src/ui/ui.css [hunk 1]
```diff
@@ -19,52 +19,10 @@ body {
...
...W5rOmhyZWY9IiNjIiBzdHJva2U9IiM0Mjg1ZjQiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDg3NDgpIi8+PC9zdmc+');background-size:100%;width:11664px;height:36px;-webkit-animation:mspin-medium-film 5332ms infinite steps(324);-moz-animation:mspin-medium-film 5332ms infinite steps(324);animation:mspin-medium-film 5332ms infinite steps(324)}@-webkit-keyframes mspin-medium-film{from{-webkit-transform:translateX(0)}to{-webkit-transform:translateX(-11664px)}}@-webkit-keyframes mspin-rotate{from{-webkit-transform:rotate(0deg)}to{-webkit-transform:rotate(360deg)}}@-webkit-keyframes mspin-revrot{from{-webkit-transform:rotate(0deg)}to{-webkit-transform:rotate(-360deg)}}@-moz-keyframes mspin-medium-film{from{-moz-transform:translateX(0)}to{-moz-transform:translateX(-11664px)}}@-moz-keyframes mspin-rotate{from{-moz-transform:rotate(0deg)}to{-moz-transform:rotate(360deg)}}@-moz-keyframes mspin-revrot{from{-moz-transform:rotate(0deg)}to{-moz-transform:rotate(-360deg)}}@keyframes mspin-medium-film{from{transform:translateX(0)}to{transform:translateX(-11664px)}}@keyframes mspin-rotate{from{transform:rotate(0deg)}to{transform:rotate(360deg)}}@keyframes mspin-revrot{from{transform:rotate(0deg)}to{transform:rotate(-360deg)}}
```
Comment (dvoytenko) [inline src/ui/ui.css]: What happened with formatting here?

Comment (dparikh) [reply src/ui/ui.css]: Well, the CSS I got is like this from : https://plus.google.com/117719742611021697975/posts/DryH7dsKPwi

Comment (dvoytenko) [reply src/ui/ui.css]: Is there some tool to format it properly?

File: src/ui/ui.css [hunk 2]
```diff
@@ -19,52 +19,112 @@ body {
...
-  left: 50% !important;
-  transform: translate(-50%, -50%) !important;
+  top: 40% !important;
+  left: 45% !important;
+  transform: translate(-40%, -40%) !important;
   z-index: 2147483647 !important;
+  width: 36px;
+  height: 36px;
+  overflow: hidden;
+  -webkit-animation: mspin-rotate 1568.63ms infinite linear;
```
Comment (dvoytenko) [inline src/ui/ui.css]: Remove prefixed versions.

File: src/ui/ui.css [hunk 3]
```diff
@@ -19,52 +19,112 @@ body {
...
...ob2Zmc2V0PSI1OS42NSIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMjczNikiIG9wYWNpdHk9Ii44Ii8+PHVzZSB4bGluazpocmVmPSIjYSIgc3Ryb2tlLWRhc2hvZmZzZXQ9IjU5LjUiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDI3NzIpIiBvcGFjaXR5PSIuODUiLz48dXNlIHhsaW5rOmhyZWY9IiNhIiBzdHJva2UtZGFzaG9mZnNldD0iNTkuNCIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMjgwOCkiIG9wYWNpdHk9Ii45Ii8+PHVzZSB4bGluazpocmVmPSIjYSIgc3Ryb2tlLWRhc2hvZmZzZXQ9IjU5LjM0IiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgyODQ0KSIgb3BhY2l0eT0iLjk1Ii8+PHVzZSB4bGluazpocmVmPSIjYSIgc3Ryb2tlLWRhc2hvZmZzZXQ9IjU5LjMyIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgyODgwKSIvPjwvZz48L2RlZnM+PHVzZSB4bGluazpocmVmPSIjYiIgc3Ryb2tlPSIjNDI4NWY0Ii8+PHVzZSB4bGluazpocmVmPSIjYyIgc3Ryb2tlPSIjZGI0NDM3Ii8+PHVzZSB4bGluazpocmVmPSIjYiIgc3Ryb2tlPSIjZGI0NDM3IiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgyOTE2KSIvPjx1c2UgeGxpbms6aHJlZj0iI2MiIHN0cm9rZT0iI2Y0YjQwMCIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMjkxNikiLz48dXNlIHhsaW5rOmhyZWY9IiNiIiBzdHJva2U9IiNmNGI0MDAiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDU4MzIpIi8+PHVzZSB4bGluazpocmVmPSIjYyIgc3Ryb2tlPSIjMGY5ZDU4IiB0cmFuc2Zvcm09InRyYW5zbGF0ZSg1ODMyKSIvPjx1c2UgeGxpbms6aHJlZj0iI2IiIHN0cm9rZT0iIzBmOWQ1OCIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoODc0OCkiLz48dXNlIHhsaW5rOmhyZWY9IiNjIiBzdHJva2U9IiM0Mjg1ZjQiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDg3NDgpIi8+PC9zdmc+');
```
Comment (dvoytenko) [inline src/ui/ui.css]: Wow! 11K!? I guess we can live with that, but this adds 30% to our binary size :)

File: src/ui/ui.css [hunk 4]
```diff
@@ -19,52 +19,112 @@ body {
...
+  background-size: 100%;
+  width: 11664px;
+  height: 36px;
+  -webkit-animation: mspin-medium-film 5332ms infinite steps(324);
```
Comment (dvoytenko) [inline src/ui/ui.css]: Ditto: no need for vendor prefixes

File: src/ui/ui.css [hunk 5]
```diff
@@ -19,52 +19,112 @@ body {
...
+  -moz-animation: mspin-medium-film 5332ms infinite steps(324);
+  animation: mspin-medium-film 5332ms infinite steps(324);
+}
+
+@-webkit-keyframes mspin-medium-film {
```
Comment (dvoytenko) [inline src/ui/ui.css]: No need for vendor-prefixed `@-webkit-keyframes` either - just leave the main one.

File: src/ui/ui.css (hunk 2 above)
Comment (dparikh) [reply src/ui/ui.css]: Removed.

File: src/ui/ui.css (hunk 3 above)
Comment (dparikh) [reply src/ui/ui.css]: May be now reduced to 10k! (since removed vendor prefixes)

File: src/ui/ui.css (hunk 4 above)
Comment (dparikh) [reply src/ui/ui.css]: Done

File: src/ui/ui.css (hunk 5 above)
Comment (dparikh) [reply src/ui/ui.css]: Done

**Pull request merged**